enduser/2021/README.md

```diff
@@ -0,0 +1,12 @@
+# CNCF End User Community Survey 2021 Results
+
+This survey was conducted with the [CNCF End User Community](https://www.cncf.io/people/end-user-community/) between August 23 - September 13, 2021.
+
+The goal was to look at the programs and ensure they meet the needs of the End User Community for 2022.
+
+The results is available in [PDF](enduser_survey_2021.pdf).
```

Comment:
Do not include the PDF as this include the total amount of responses. This is not shown in previous results. Follow the format for previous years e.g https://github.com/cncf/surveys/tree/master/enduser/2020

Reply:
Oh ok, I will remove it.